src/Apps/CollectorProfile/Routes/Saves2/Components/Actions/Mutations/useCreateCollection.ts

```diff
@@ -0,0 +1,68 @@
+import { ConnectionHandler, graphql } from "react-relay"
+import { RecordSourceSelectorProxy } from "relay-runtime"
+import { useMutation } from "Utils/Hooks/useMutation"
+import {
+  useCreateCollectionMutation,
+  useCreateCollectionMutation$data,
+} from "__generated__/useCreateCollectionMutation.graphql"
+
+const onListAdded = (
+  store: RecordSourceSelectorProxy<useCreateCollectionMutation$data>,
+  data
```

Comment:
**suggestion**: `data: useCreateCollectionMutation$data`

Reply:
If I do that then store is not the correct type and we don't have access to `getRoot` and `getRootField` and we're getting this error

```
Property 'getRoot' does not exist on type 'useCreateCollectionMutation$data'.ts(2339)
```

Reply:
ooooh, ignore my comment I realized that it was for data 📦